### DISCUSSION TITLE
Can I use pygame ?

### DISCUSSION THREAD

**Discussion by jero7** [exodrifter/unity-python#19] [Q&A]
Hi, I want to ask if it's possible to use pygame in this porject or not. I'm trying to get data from python code to unity but it gives me error like "ImportException: No module named pygame" Thanks

**Comment by exodrifter** [comment]
I'm not sure if you can use pygame or not. I would guess that you can't, but you could certainly try. If you could come back and post about your experience, it would help others who have the same concern and I'd really appreciate the additional information.

As for your specific error, it sounds like you didn't configure things correctly. Did you [update the configuration](https://github.com/exodrifter/unity-python/blob/2e0bc6834c0b8769d84f69487e3b64b4803d8774/Examples/PythonLibrary.cs#L18) to use the right search paths? If you could be more detailed about the *what* and *why* of what you're trying to do it could help me give you more useful help.

**Comment by exodrifter** [comment]
@jero7 Feel free to reopen this ticket if you still need help and can provide the information I asked for.
